### ISSUE TITLE
LOGO

### ISSUE THREAD

**Issue by heaucques** [Orbiit/gunn-remake#5]
HOW DO WE INCORPORATE A LOGO
ALSO WHAT SHOULD THE LOGO BE

**Comment by SheepTester** [comment]
Maybe we can have a minimalist version of Titan the Saturn moon over the bottom half of a circle (eg semicircle with flat side up) for a "G". We can continue to use semicircles (and rectangles?) for the U and N; the N can just be an upside down u and vice versa.

It needs to be in two forms and two forms. One of the two forms is just the special circle-and-semicircle G. It would be supposedly used:

- as the favicon
- that's it. ._.

The other form is the first form plus the `unn`. It'll be used in:

- the navbar
- where else are website logos put? ._.

There's also the two forms: the coloured version and the silhouette version. The coloured version should only be used on a white/black background or else you don't deserve to be a person.

Maybe I can make the silhouette version by typing up some SVG happiness _morgaŭ_ but not the coloured version as I don't know how to make a happy Titan

**Comment by heaucques** [comment]
oh dears that alot to read

**Comment by heaucques** [comment]
I tried without the moon design but it looks a little odd
I can try though

**Comment by PooleHenry** [comment]
The other official gunn logo is pretty minimalist. I've got a png version: ![here](http://i.imgur.com/CqW5nHU.png) and a vector version.

**Comment by SheepTester** [comment]
It's terrible. It looks like someone stretched it. And it's a generic G; it represents nothing about what Gunn is. It also isn't everywhere on the Gunn website, so it isn't easily recognizable.

And the red!

Also, I don't want to bother with copyright issues, and it's better if we remake everything anyways.

Worst case scenario we just make a simplistic red geometric capital G.

**Comment by heaucques** [comment]
Even worse case scenario we use MS Paint

**Comment by SheepTester** [comment]
> _@Velosify wrote:_
> Even worse case scenario we use MS Paint

to make a simplistic red geometric capital G

**Comment by PooleHenry** [comment]
I made this one in Google drawings, vectorized it, and set its color to the one used on the current Gunn website. Let me know what you guys think. (jpg screenshot is attached)

![screen shot 2017-09-01 at 6 17 28 pm](https://user-images.githubusercontent.com/29491154/29991772-dfdb1372-8f41-11e7-96a7-de0c16f9303f.png)

This probably doesn't fit the site look, though. 👍

**Comment by heaucques** [comment]
@PooleHenry Looks nice but overused concept

**Comment by SheepTester** [comment]
I disagree; it looks nice.

It could be cleaned up a bit, and the red could be changed to something else. It's better than Gunn's current logo at least

**Comment by heaucques** [comment]
@SheepTester How do you disagree about it looking nice when we agree? Im confused :/

**Comment by SheepTester** [comment]
I disagree with the "but" used in your previous post that indicated that being overused is bad

"Overused" in general has a negative connotation

**Comment by heaucques** [comment]
Yes it does but I don't oppose the concept

**Comment by PooleHenry** [comment]
I'm going to clean up the logo today. I'm going to round up that intersection between the G and the top line, as it really sticks out at me. I can also make some other changes while I'm at it if you guys say so.

**Comment by SheepTester** [comment]
![image](https://user-images.githubusercontent.com/22133785/30040176-79417b00-9190-11e7-8775-2b066c95c633.png)
Perhaps?

(SVG version available [in the repository](https://github.com/Orbiit/gunn-remake/blob/master/logoconcepts/henrygunnlogo.svg))

**Comment by PooleHenry** [comment]
Whoa nice!

**Comment by MajikalExplosions** [comment]
Güd Jobs.
I was gone for a bit so I couldn't do much work.
And why is the color of your new Gunn logo the same as your sheep logo?

**Comment by SheepTester** [comment]
I forgot to change the fill colour

fortunately it's an SVG so it shouldn't be too hard to change it

**Comment by SheepTester** [comment]
![image](https://user-images.githubusercontent.com/22133785/30221378-d9d19de4-9477-11e7-8416-7bce3ec1db08.png)
We can use this as a favicon; I've put the SVG [in the repository](https://github.com/Orbiit/gunn-remake/blob/master/logoconcepts/henrygunnlogomini.svg) también.

Of course, this doesn't address any of the issues I've listed before
